argocd/overlays/moc-infra/projects/otel-dev.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: hello-opentelemetry
```

Comment:
did you mean to name this argocd project `octo-otel-dev`?

Reply:
yes - I was going to name it  hello-opentelemetry project but was confused about whether it should match what already exists. What's the meaning of the `otel-dev-*` here? Will other namespaces/projects be created? 
```yaml
spec:
  destinations:
    - namespace: 'otel-dev-*'
      server: 'https://api.smaug.na.operate-first.cloud:6443
```

Reply:
for now I changed it to `octo-otel-dev` that works.
I installed the helm charts already in otel-dev ns - that's what is running in there now